06-building-a-program-start-to-finish/writing-tests/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: Mocha/Jasmine with any assertion library of choice (Chai.JS, Should.JS, Expect.JS...)
+- **Deliver**: a Pull Request showing all tests required for your project
+- **With**: *independently*
+
+#### Targeted Learning Outcomes
```

Comment:
@q-ode This does not say/assess end to end testing at all! We need them to know about that Selenium goodness!